### PR TITLE
ci: preserve infrastructure

### PR DESCRIPTION
**Pull request by rajkarkar-crest**
This is comment
PR body line 1
PR body line 2
preserve: knowledge

### CODE REVIEW

**Comment by rajkarkar-crest** [comment]
rerun: ui

**Comment by rajkarkar-crest** [comment]
rerun: ui

**Comment by rajkarkar-crest** [comment]
rerun: ui

**Comment by rajkarkar-crest** [comment]
ci: rerun

**Comment by truptilangalia-crest** [comment]
looks good

File: .github/workflows/build-test-release.yml [hunk 1]
```diff
@@ -6,13 +6,22 @@ on:
       - "develop"
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+"
-  pull_request:
```
Comment (truptilangalia-crest) [inline .github/workflows/build-test-release.yml:9]: looks good

**Pull request closed without merging**